src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,58 @@
+.project-card {
+    height: 200px;
+    width: 300px;
+    border: 1px solid rgb(213, 213, 213);
+    position: relative;
+
+    h3 {
+        text-align: center;
+    }
+}
+
+.project-bg {
+    background: linear-gradient(to right, rgba(255, 0, 0, 0.7), rgba(0, 128, 0, 0.5), rgba(0, 0, 255, 0.7), rgba(128, 0, 128, 0.7), rgba(0, 0, 0, 0.7), rgba(245, 241, 4, 0.7)), url("../../img/project-screenshots/city-info-screenshot.png") center center no-repeat;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)